atxsdata/warmup_test.go

```diff
@@ -74,14 +74,6 @@ func TestWarmup(t *testing.T) {
 		require.Error(t, err)
 		require.Nil(t, c)
 	})
-	t.Run("missing nonce", func(t *testing.T) {
-		db := sql.InMemory()
-		data := gatx(types.ATXID{1, 1}, 1, types.NodeID{1}, nil)
-		require.NoError(t, atxs.Add(db, &data))
-		c, err := Warm(db, 1)
-		require.Error(t, err)
-		require.Nil(t, c)
-	})
```

Comment:
The majority of existing ATXs does not have a VRFNonce in them

Reply:
Restored the test, it was removed by mistake